libs/vm_monitor/src/cgroup.rs

```diff
@@ -232,6 +250,24 @@ impl MemoryStatus {
 
         DS(slice)
     }
+
+    /// Check if the other memory status is a close or similar result.
+    /// Returns true if the larger value is not larger than the smaller value
+    /// by 1/8 of the smaller value, and within 128MiB.
+    /// See check_similarity_behaviour for examples of behaviour
```

Comment:
You can make this a link in rustdoc: https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html